### PR TITLE
TOC laplan update

### PR DESCRIPTION
**Pull request by tiffanychu90**
* Update notebooks related to TOC analysis to use `laplan` and make sure results are the same
* Update docs 
* Fix and add `parsed_zoning` as zipped shapefile to catalog.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +31,7 @@ Land use permitting analysis for Department of City Planning
     ├── requirements.txt         <- The requirements file for reproducing the analysis environment, e.g.
     │                               generated with `pip freeze > requirements.txt`
     │
-    ├── setup.py                 <- makes project pip installable (pip install -e .) so src can be imported
+    ├── setup.py                 <- Makes project pip installable (pip install -e .) so src can be imported
```
Comment (ian-r-rose) [inline README.md:34]: Should have caught this earlier. This `setup.py` is now for `laplan`, and the scripts in `src` are not installed/importable this way any more.

File: notebooks/C1-build-toc-master.ipynb [hunk 2]
```diff
@@ -58,16 +58,19 @@
...
     "        prefix_list=prefix,\n",
-    "        suffix_list=suffix,\n",
     "        get_dummies=True,\n",
     "    )\n",
-    "        \n",
+    "    \n",
+    "    pcts = laplan.pcts.drop_child_cases(pcts, keep_child_entitlements = True)\n",
+    "    \n",
+    "    dropme = prefix_suffix_cols = prefix + list(laplan.pcts.VALID_PCTS_SUFFIX)\n",
+    "    pcts = pcts.drop(columns = dropme)\n",
```
Comment (ian-r-rose) [inline notebooks/C1-build-toc-master.ipynb:72]: Why is this necessary? Is there a problem with the exclusion of prefixes/suffixes above?

**Review by ian-r-rose — APPROVED** [review]
Thanks @tiffanychu90!

File: notebooks/C1-build-toc-master.ipynb (hunk 2 above)
Comment (tiffanychu90) [reply notebooks/C1-build-toc-master.ipynb:72]: Ahh, it's a very nuanced point. But, they wanted to know if parcels within TOC Tiers had a lot of non-TOC activity too. Like, yes, Tier 3 might have a lot of TOC entitlements, but do they also have a lot of non-TOC entitlements happening? 

Once I know which parcels fall within TOC Tiers, those parcels are such a small fraction of parcels within the city. It winnows down the non-TOC entitlements that are happening in that same period of 10/2017-onward. 

My charts depend on counting how many TOC vs non-TOC entitlements occurred. The dummies aren't important, since I want to squash all the other suffixes into non-TOC.

Comment (ian-r-rose) [reply notebooks/C1-build-toc-master.ipynb:72]: Ah, got it. Thanks!
